### PR TITLE
add --version

### PR DESCRIPTION
**Pull request by wagner-certat**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -57,6 +57,7 @@
     install_requires=[
         'pygpgme>=0.3',
         'psycopg2',
+        'pyxarf',
```
Comment (bernhardreiter) [inline setup.py]: Right now the pypi verison of pyxarg 0.5.0 will probably not work, but the master version would.
In order to get a way to specify the dependency more properly, I've suggested to "release" a new version of pyxarf:
https://github.com/xarf/python-xarf/pull/2

Thanks @wagner-certat for pointing out that this dependeny is missing.

Comment (wagner-certat) [reply setup.py]: Specifying the direct git/github url does not work inside the setup.py parameters. I tried several possibilities, but nothing worked. So it needs to be documented "manually".

Comment (bernhardreiter) [reply setup.py]: Added manual documentation with https://github.com/Intevation/intelmq-mailgen/commit/7957676c83d32e40daa31b53ec6e83a34496816a Thanks @wagner-certat"

**Comment by bernhardreiter** [comment]
If you want the pull request to go in directly it would need to be updated to only include the `--version` functionality. (I've already edited the title accordingly.)

**Comment by wagner-certat** [comment]
> What do you need the `--version` attribute for? (If you want to check which code is actually run, then using `pkg_resources` may not be the best solution if I understand the overview correctly.)

No strong need behind it. Usually executables have this parameter and I tried it during testing, so I just added it.

**Pull request closed without merging**